src/test/kotlin/com/atlassian/performance/tools/virtualusers/api/EntryPointIT.kt

```diff
@@ -1,34 +1,101 @@
 package com.atlassian.performance.tools.virtualusers.api
 
-import com.atlassian.performance.tools.dockerinfrastructure.api.jira.Jira
-import com.atlassian.performance.tools.dockerinfrastructure.api.jira.JiraCoreFormula
+import com.atlassian.performance.tools.infrastructure.api.database.MySqlDatabase
+import com.atlassian.performance.tools.infrastructure.api.dataset.Dataset
+import com.atlassian.performance.tools.infrastructure.api.dataset.HttpDatasetPackage
+import com.atlassian.performance.tools.infrastructure.api.jira.JiraHomePackage
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
 import com.atlassian.performance.tools.virtualusers.ChromeContainer
+import com.atlassian.performance.tools.virtualusers.DockerJira
 import com.atlassian.performance.tools.virtualusers.SimpleScenario
-import org.junit.Ignore
+import com.atlassian.performance.tools.virtualusers.api.ActivityType.*
+import com.atlassian.performance.tools.virtualusers.lib.infrastructure.Jperf425WorkaroundMysqlDatabase
+import org.assertj.core.api.Assertions.assertThat
 import org.junit.Test
+import java.net.URI
+import java.nio.file.Paths
+import java.time.Duration
+import java.util.*
+import java.util.stream.Stream
+import kotlin.streams.asSequence
 
 class EntryPointIT {
 
-    @Ignore("https://github.com/testcontainers/testcontainers-java/issues/1113")
```

Comment:
We could jut unignore when https://github.com/testcontainers/testcontainers-java/pull/2882 is released in testcontainers 1.15.0

Reply:
Done in https://github.com/atlassian/virtual-users/pull/44